### ISSUE TITLE
[Feature request] Vsync isn't smooth on 120hz screen

### ISSUE THREAD

**Issue by Immersion95** [RPCS3/rpcs3#12578]
### Quick summary

RPCS3 has a lot of judder on 120hz screen but works perfectly on 60hz screen. Adding an option for Vsync swap interval would be great.

### Details

There's no tearing with Vsync but the experience isn't smooth at all.

**Comment by EdHerdman** [comment]
180Hz option would be welcome too.

**Comment by Augusto7743** [comment]
All right technology need progress, but 240 HZ not is much high ? Really is an good improvement for gameplay ?

**Comment by digant73** [comment]
@kd-11 maybe with vsync and framelimit settings (both present now also on home menu so hot-reloadable) this FR is implemented and so this ticket can be closed?

**Comment by EdHerdman** [comment]
I've never gotten the framelimit settings working to my liking, but do those settings explicitly allow you to set 60FPS/60Hz and 60FPS/120Hz, for example?

**Comment by kd-11** [comment]
Framelimit off + vsync full will give you good framepacing. The problem with CPU framelimit is that it is cursed to create bubbles especially since emulated games don't actually run at arbitrary framerates. I think the current implementation should be good enough to avoid judder.

**Comment by Immersion95** [comment]
Totally forgot it. Using Frame limiter Auto + Vsync Off gives perfect framepacing with VRR/Freesync.

**Comment by digant73** [comment]
yeah, I think the OP tried to play the game at 120 fps (probably using vsync to match (or avoid tearing) 120Hz TV output) but fps was not a solid 120fps so he was asking to sync on a stable 60fps on a 120hz tv output. And that is possible with framelimiter set to 60 (and eventually vsync enabled to avoid also tearing)

EDIT: OP replied while I was also replying
